### ISSUE TITLE
NameError: name 'TF_MLC_LOGGING' is not defined

### ISSUE THREAD

**Issue by walleym** [apple/tensorflow_macos#160]
The repo readme talks about using certain environment variables such as 'TF_MLC_LOGGING' and 'TF_DISABLE_MLC_EAGER'. However when I use them in my project they cannot be found. I am running on a 2020 macbook pro with an M1 chip and my tensorflow is running through a virtual environment. Could someone explain how to use these environment variables. It would also be useful to know how to switch to eager mode.

**Comment by pooyadavoodi** [comment]
Thanks for trying this out.

Could you post how you use `TF_MLC_LOGGING`?

Typically there are two ways of setting an env var:
1) Set it in terminal before running your python script:
```
% export TF_MLC_LOGGING=1
% python my_script.py
```
2) Set it inside your python script:

```
# This is the top of the script.
# A bunch of imports appear at the top.

# Now set the env var:
os.environ["TF_MLC_LOGGING"] = "1"

# The rest of the script comes here.
```

------------

If you run your script with setting the env var `TF_MLC_LOGGING` (using any of the above methods), then you should see these two lines at the top of the log. Please let me know if you can see these logs.:
```
... I ./tensorflow/compiler/tf2mlcompute/convert/mlc_graph_conversion_pass.h:35] MLCGraphConversionPass: Initialized
... I tensorflow/compiler/tf2mlcompute/eager/mlc_eager_op_rewrite.cc:45] MLCEagerOpRewrite: Initialized
```

**Comment by walleym** [comment]
Ok I tried setting them like that. I do not get those logs in the console. I am setting "TF_DISABLE_MLC_EAGER"] to include "MLCMatMul". This is because I want to load the model on a non-ARM mac and this op doesn't work on normal tensorflow. This does not work, my normal tensorflow still fails trying to find that op. Is there a way of setting it to run fully in eager mode with no MLcompute ops?

**Comment by anna-tikhonova** [comment]
Did you train your model with our fork on a M1 Mac and now trying to run inference using TensorFlow without ML Compute acceleration (i.e. without our fork installed on your system)? Could you please provide a reproducible test case for us to run locally? It would be very helpful for us to be able to run the exact code you are running.

**Comment by walleym** [comment]
Yes I created the model on the ML Compute optimised version. I saved the model and now I want to run it on a mac which is running normal tensorflow without the ML Compute optimisation available. I think this should be applicable to any model that you create. For example we could use a simple mnist model. I would like to disable all ML Compute ops so that the saved model is portable to normal tensorflow versions. I hope this helps clarify.

**Comment by walleym** [comment]
TF_MLC_LOGGING does work if I set it through export in the command line but not if I do it in the script. Any reason why that is?
